pkg/volume/util/device_util_linux.go

```diff
@@ -99,7 +99,7 @@ func (handler *deviceHandler) GetISCSIPortalHostMapForTarget(targetIqn string) (
 	sysPath := "/sys/class/iscsi_host"
 	hostDirs, err := io.ReadDir(sysPath)
 	if err != nil {
-		return nil, err
+		return portalHostMap, nil
```

Comment:
We need to distinguish between does-not-exist and cant-read types of error here. It's only correct to return the empty map when the directory doesn't exist.

Reply:
thanks for your review,and i have remodify it for your review!